packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -49,7 +58,7 @@ export const chatMessageStyles: ComponentSlotStylesPrepared<ChatMessageStylesPro
           }),
         }),
 
-      ...(p.compact ? chatMessageStylesCompact : chatMessageStylesComfy).root(componentStyleFunctionParam),
+      ...chatMessageDensityStyles[p.density].root?.(componentStyleFunctionParam),
```

Comment:
`p.density` is not guaranteed to be defined, it would be good set a default value or wrap it in a function call like in this suggestion.

```suggestion
      ...getChatMessageDensityStyles(p.density).root?.(componentStyleFunctionParam),
```

Reply:
Added a getter function:
```ts
const getChatMessageDensityStyles = (density: ChatDensity = defaultChatDensity) => chatMessageDensityStyles[density];
```